_data/transport.yml

```diff
@@ -165,6 +165,13 @@ websites:
       img: sas.png
       tfa: No
 
+    - name: Scotrail
```

Comment:
Their name appears to be stylized as "ScotRail" according to their website and social media profiles.
Please update their name to be `ScotRail`.